<h1 align="center">Felipe Bitencourt Martins</h1>

<h3 align="center"><i>PhD student in Genetics and Molecular Biology, Bioinformatician and Data Scientist</i></h3>

<div align="center">

[![Linkedin Badge](https://img.shields.io/badge/LinkedIn-0077B5?style=flat&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/felipe-martins-6a36b9174/)&nbsp;&nbsp;
[![Medium Badge](https://img.shields.io/badge/Medium-12100E?style=flat&logo=medium&logoColor=white)](https://medium.com/@febimartins)&nbsp;&nbsp;
[![Twitter Badge](https://img.shields.io/badge/Twitter-1DA1F2?style=flat&logo=twitter&logoColor=white)](https://twitter.com/Felitencourt)&nbsp;&nbsp;
[![Instagram Badge](https://img.shields.io/badge/Instagram-E4405F?style=flat&logo=instagram&logoColor=white)](https://www.instagram.com/bitafelipe/)&nbsp;&nbsp;
[![Gmail Badge](https://img.shields.io/badge/Gmail-D14836?style=flat&logo=gmail&logoColor=white&link=mailto:febimartins@gmail.com)](mailto:febimartins@gmail.com)&nbsp;&nbsp;
</div>

<div style="text-align: justify"> Realizou um est√°gio no Laborat√≥rio de Gen√¥mica e Prote√¥mica da Unicamp no primeiro semestre de 2012, foi monitor de Biologia (bolsa PADEMT) durante o segundo semestre de 2012 no Col√©gio T√©cnico da Unicamp(COTUCA), cursou 2 semestres de Ci√™ncias Biol√≥gicas na Universidade Federal de Uberl√¢ndia(UFU) onde foi bolsista do programa Jovens Talentos para a Ci√™ncia, se formou como bacharel em Ci√™ncias Biol√≥gicas pela Universidade Estadual de Campinas(Unicamp) em 2017 e atualmente est√° cursando doutorado direto em Gen√©tica e Biologia Molecular na mesma institui√ß√£o . Durante a gradua√ß√£o realizou Inicia√ß√£o Cient√≠fica no Laborat√≥rio de An√°lise Gen√©tica e Molecular da Unicamp recebendo bolsa FAPESP, tendo como objetivo identificar e caracterizar genes candidatos a c√≥pia √∫nica em cana-de-a√ß√∫car e no doutorado, no mesmo laborat√≥rio, trabalha com esp√©cies de brachiaria com o objetivo de analisar transcriptomas, construir mapas gen√©ticos e estimar GEBVs para sele√ß√£o gen√¥mica.
</div>

### Analytics Tools

* **Data Collection and Storage:** SQL.

* **Data Processing and Analytics:** Python and R.

* **Data Visualization:** ggplot2, Seaborn, Matplotlib.

* **Machine Learning Modeling:** Classification, Regression and Clustering.

## Projects

* ### [An Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.biorxiv.org/content/10.1101/2021.07.01.450796v1.full)

    Artificial hybridization plays a fundamental role in plant breeding programs since it generates new genotypic combinations that can result in desirable phenotypes. Depending on the species and mode of reproduction, controlled crosses may be challenging, and contaminating individuals can be introduced accidentally. In this context, the identification of such contaminants is important to avoid compromising further selection cycles, as well as genetic and genomic studies. The main objective of this work was to propose an automated multivariate methodology for the detection and classification of putative contaminants, including apomictic clones, self-fertilized individuals, half-siblings and full contaminants, in biparental polyploid progenies of tropical forage grasses. We established a pipeline to identify contaminants in **genotyping-by-sequencing (GBS)** data encoded as **allele dosages of single nucleotide polymorphism (SNP)** markers by integrating **principal component analysis (PCA)**, genotypic analysis (GA) measures based on Mendelian segregation and **clustering analysis (CA)**. The combination of these methods allowed the correct identification of all contaminants in all simulated progenies and the detection of putative contaminants in three real progenies of tropical forage grasses, providing an easy and promising methodology for the identification of contaminants in biparental progenies of tetraploid and hexaploid species. The proposed pipeline was made available through the <a href="https://github.com/lagmunicamp/polycid" target="_blank">**polyCID**</a> Shiny app and can be easily coupled with traditional genetic approaches, such as linkage map construction, thereby increasing the efficiency of breeding programs.

    
* ### [Fraudulent Transaction Detection](https://github.com/juniorcl/transaction-fraud-detection)

    Financial transactions fraud is one of the biggest problems faced by financial institutions. Thus, this project uses data science and machine learning to detect and avoid fraudulent transactions. The model got a precision of 96.3% +/- 0.7% and a recall of 76.3% +/- 3.5%. The profit expected by the company is R\$ 57,251,574.44.

</div>

<!--
**felipebita/felipebita** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
