<h1 align="center">Felipe Bitencourt Martins</h1>

<h3 align="center"><i>PhD student in Genetics and Molecular Biology, Bioinformatician and Data Scientist</i></h3>

<div align="center">

[![Linkedin Badge](https://img.shields.io/badge/LinkedIn-0077B5?style=flat&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/felipe-martins-6a36b9174/)&nbsp;&nbsp;
[![Medium Badge](https://img.shields.io/badge/Medium-12100E?style=flat&logo=medium&logoColor=white)](https://medium.com/@febimartins)&nbsp;&nbsp;
[![Twitter Badge](https://img.shields.io/badge/Twitter-1DA1F2?style=flat&logo=twitter&logoColor=white)](https://twitter.com/Felitencourt)&nbsp;&nbsp;
[![Instagram Badge](https://img.shields.io/badge/Instagram-E4405F?style=flat&logo=instagram&logoColor=white)](https://www.instagram.com/bitafelipe/)&nbsp;&nbsp;
[![Gmail Badge](https://img.shields.io/badge/Gmail-D14836?style=flat&logo=gmail&logoColor=white&link=mailto:febimartins@gmail.com)](mailto:febimartins@gmail.com)&nbsp;&nbsp;
</div>

## Education and Research
I have a bachelor's degree in Biological Science (State University of Campinas - UNICAMP) and I'm currently working on my PhD in [Genetics and Molecular Biology](https://site2.ib.unicamp.br/pos_genetica_molec/en/node/129) (State University of Campinas - UNICAMP). As an undergraduate student I had a FAPESP research scholarship to accomplish a project in the [Laboratory of Genetic and Molecular Analysis](https://lagm.cbmeg.unicamp.br/en/index.html) where I investigated single copy genes in sugarcane using bioinformatic tools. In the same laboratory I started my PhD studying plant breeding, more specifically molecular breeding methods and tools for polyploid species. During this time I acquired abilities to obtain, manipulate and analyze genetic and common data using programming and statistics. Furthermore, I have been studying Machine Learning algorithms to enhance plant breeding results and resolve business problems.

#### Tools
* **Operating system:** Unix(Linux, centOS, macOS) and Windows

* **Data Processing, Analytics and Visualization:** SQL, Python and R.

* **Machine Learning Modeling:** Classification, Regression and Clustering.

## Projects

* ### [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full)

    Artificial hybridization plays a fundamental role in plant breeding programs since it generates new genotypic combinations that can result in desirable phenotypes. Depending on the species and mode of reproduction, controlled crosses may be challenging, and contaminating individuals can be introduced accidentally. In this context, the identification of such contaminants is important to avoid compromising further selection cycles, as well as genetic and genomic studies. The main objective of this work was to propose an automated multivariate methodology for the detection and classification of putative contaminants, including apomictic clones, self-fertilized individuals, half-siblings and full contaminants, in biparental polyploid progenies of tropical forage grasses. We established a pipeline to identify contaminants in **genotyping-by-sequencing (GBS)** data encoded as **allele dosages of single nucleotide polymorphism (SNP)** markers by integrating **principal component analysis (PCA)**, genotypic analysis (GA) measures based on Mendelian segregation and **clustering analysis (CA)**. The combination of these methods allowed the correct identification of all contaminants in all simulated progenies and the detection of putative contaminants in three real progenies of tropical forage grasses, providing an easy and promising methodology for the identification of contaminants in biparental progenies of tetraploid and hexaploid species. The proposed pipeline was made available through the <a href="https://github.com/lagmunicamp/polycid" target="_blank">**polyCID**</a> Shiny app and can be easily coupled with traditional genetic approaches, such as linkage map construction, thereby increasing the efficiency of breeding programs.

    
* ### [Fraudulent Transaction Detection](https://github.com/felipebita/fraud_detection)
    Financial institutions are facing problems when dealing with fraudulent transactions. Those transactions not only reduces the institution revenue but also its credibility. In this scenario, the Block Fraud Company offers a service that ensures the blocking of fraud transactions. The methodology developed by the BFC is based on machine learning algorithms for classification, the implemented model has a precision of 0.958 +/- 0.013 and a recall of 0.744 +/- 0.037. Considering the company expansion strategy and the model capacity, for a set of 120.000 transactions, the service would generate a total revenue of $39.309.131, with a mean profit of $330,00 per transaction analyzed.
    
## Complementary Education
* **(2021) Genomic Selection.** Conecta GEM, Brasil. (6h)
* **(2021) Python for Natural Language Processing (NLP): ICMC 50th Anniversary Edition.** Instituto de Ci√™ncias Matem√°ticas e de Computa√ß√£o, USP, Brasil (20 hours).
* **(Course Assistant) (2021) Introduction to Python Programming.** Grupo de Estudos em Gen√©tica e Melhoramento (GenMelhor), UFV, Brasil (16 hours).
* **(Course Assistant) (2021) Introduction to Python Programming.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (16 hours).
* **(2021) Introduction to Machine Learning in Python.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **(2021) Spring Workshop on Applied Quantitative Genetics for Plant Breeders.** University of Illinois at Urbana-Champaign, USA.
* **(2021) Tools for Genomics-Assisted Breeding in Polyploids: Workshop.** Development of a Community Resource, National Institute of Food and Agriculture, USDA.
* **(2019) Polyploid Genetic Data Analysis: From Dosage Calling To Linkage.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (21 hours).
* **(2018) Statistical Mechanisms and Practical Applications of Genome-Wide Associatio.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **(2018) Prediction of Breeding Values in Autotetraploid Forages.** Empresa Brasileira de Pesquisa Agropecu√°ria, EMBRAPA, Brasil (32 hours).
* **(2018) Python for Genomic Data Science by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **(2018) Getting and Cleaning Data by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **(2018) R Programming by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **(2017) Bioinformatic Winter Course: Workshop.** Universidade Federal de S√£o Paulo, UNIFESP, Brasil (22 hours).
* **(2017) Bioinformatic Analysis of NGS Data.** Sociedade Brasileira de Gen√©tica, SBG, Brasil (3 hours).

## Lectures and Presentations
* ### [Genetic Markers](https://drive.google.com/file/d/1jXof4E6BPAY00_-0V_jWP3NAH1zdsNeK/view?usp=sharing) (slides)(pt-br)
    This slide presentation was used in a lecture ministered in the [BV880 - Plant Breeding - 2S/2019](https://www.dac.unicamp.br/portal/caderno-de-horarios/2019/2/S/G/IB/BV880) class for the Biological Sciences course at UNICAMP. It is an introduction about markers (morphological and molecular) with focus on molecular markers,especially the DNA markers. It covers the most common markers (microsatellite and SNP) and it's use in linkage maps and QTL mapping.
    
* ### [Genomic Selection](https://drive.google.com/file/d/1mCsgztu_3Aw5fAifUz7Rt2LNbj2TSLjq/view?usp=sharing) (video)(pt-br)
    This lecture was ministered in the [BV782 - Plant Genetics and Genomics - 1S/2020](https://www.dac.unicamp.br/portal/caderno-de-horarios/2020/1/S/G/IB/BV782) class for the Biological Sciences course at UNICAMP. It is an introduction about Genomic Selection, passing through the famous Meuwissen (2001) article which demonstrate the applicability of dense sets of genetic markers to predict breeding values, how Genomic Selection models are evaluted (Predictive Ability and Cross Validation) and part of my research regarding genomic selection in half-siblings families of *Urochloa ruziziensis* tropical forage.

* ### [Linear Regression](https://docs.google.com/presentation/d/1R6fViHULsb0LGOy7FC9hUny7v4ktDfaD/edit?usp=sharing&ouid=106210785402829867900&rtpof=true&sd=true) (slides)(en)
    This presentation was constructed as a scheme for my own studies and latter used to teach lab colleagues. It covers the three main topics about Linear Regressions: (1) fit a line to the data; (2) calculate the R¬≤ and (3) calculate de F-test.
    
* ### [Contaminant Identification in Polyploids](https://drive.google.com/file/d/1K8t9R4DzqFK9VmUHQcnjwUwpBBbcKFyv/view?usp=sharing) (video)(en)
    Video-poster of the article [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full) submited in the [11¬∫ CBPM - Brazilian Congress of Plant Breeding](https://www.cbmp2021.com.br/).
    
    
</div>

<!--
**felipebita/felipebita** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
