<h1 align="center">Felipe Bitencourt Martins</h1>

<h3 align="center"><i>PhD student in Genetics and Molecular Biology, Bioinformatician and Data Scientist</i></h3>

<div align="center">

[![Linkedin Badge](https://img.shields.io/badge/LinkedIn-0077B5?style=flat&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/febimartins/)&nbsp;&nbsp;
[![Medium Badge](https://img.shields.io/badge/Medium-12100E?style=flat&logo=medium&logoColor=white)](https://medium.com/@febimartins)&nbsp;&nbsp;
[![Twitter Badge](https://img.shields.io/badge/Twitter-1DA1F2?style=flat&logo=twitter&logoColor=white)](https://twitter.com/Felitencourt)&nbsp;&nbsp;
[![Instagram Badge](https://img.shields.io/badge/Instagram-E4405F?style=flat&logo=instagram&logoColor=white)](https://www.instagram.com/bitafelipe/)&nbsp;&nbsp;
[![Gmail Badge](https://img.shields.io/badge/Gmail-D14836?style=flat&logo=gmail&logoColor=white&link=mailto:febimartins@gmail.com)](mailto:febimartins@gmail.com)&nbsp;&nbsp;
</div>

## Education and Research
I have a [bachelor's degree](https://drive.google.com/file/d/1pI8eVAobot_1tvGbaeYL_SpZDoFVsdbc/view?usp=sharing) in Biological Sciences (State University of Campinas - UNICAMP) and I'm currently working on my PhD in [Genetics and Molecular Biology](https://site2.ib.unicamp.br/pos_genetica_molec/en/node/129) (State University of Campinas - UNICAMP). As an undergraduate student I had a FAPESP research scholarship to accomplish a project in the [Laboratory of Genetic and Molecular Analysis](https://lagm.cbmeg.unicamp.br/en/index.html) where I investigated single copy genes in sugarcane using bioinformatic tools. In the same laboratory I started my PhD studying plant breeding, more specifically molecular breeding methods and tools for polyploid species. During this time I acquired abilities to obtain, manipulate and analyze genetic and common data using programming and statistics. Furthermore, I have been studying Machine Learning algorithms to enhance plant breeding results and resolve business problems.

## Skills
        Data Science                                     Research                                           Microsoft Windows
        Data Analysis                                    Bioinformatics                                     Apple Mac
        Data Mining                                      Computational Biology                              Linux
        Data Visualization                               Genetics                                           Ubuntu
        Statistics                                       Plant Genetics                                     CentOS
        Machine Learning                                 Plant Breeding
        Python                                           Genomic Data
        R                                                Genotyping
        SQL                                              Molecular Markers
        GitHub                                           GBS
        AWS                                              Transcriptome
        Microsoft PowerPoint                             Simulation
        Microsoft Excel                                  Teaching
        Microsoft Word                                   
        
## Scientific Production and Projects

* ### [Insights into the Regulation of Agronomic Traits in _U. ruziziensis_ Through Multi-omics Data Integration](https://drive.google.com/file/d/1EQoMBI_S97q8Ij12aUPFb-UAPekSEIcZ/view?usp=sharing)

    The complete manuscript is not finished, it will be available when submited as pre-print to bioRxiv.
    Tropical forage grasses, especially species of the genus Urochloa, play an important role in cattle production being the main food source for the animals in the tropical/subtropical regions. Most of the species are apomictic and tetraploid, this give special importance to U. ruziziensis, a sexual diploid species that can be tetraploidized to be used in interspecific crosses with apomitic species. As a means to auxiliate breeding programs, this study investigates the applicability of **genome wide family prediction (GWFP)** in U. ruziziensis half-sibling families to predict growth and biomass production. **Machine learning** and **feature selection algorithms** were used to reduce the number of markers necessary for prediction to approximately 11 markers and to enhance the predictive ability to approximately 0.9 across the phenotypes. Beyond that, to investigate the regulation of the agronomic traits, the position of the markers with more importance for the prediction were considered as putative QTLs, and in a **multi-omic approach**, genes obtained in the species transcriptome were mapped linked to those markers. Furthermore, a **gene co-expression network** was constructed enabling not only the investigation of the mapped genes but their co-expressed genes too. The functional annotation showed that the mapped genes are mainly associated with auxin transport and biosynthesis of lignin, flavonol and folic acid, while the co-expressed genes are associated with DNA metabolism, stress response and circadian rhythm. These results provide a viable marker-assisted breeding approach for tropical forages and identify target regions for future molecular studies on agronomic traits.
        
* ### [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full)

    Artificial hybridization plays a fundamental role in plant breeding programs since it generates new genotypic combinations that can result in desirable phenotypes.  Depending on the species and mode of reproduction, controlled crosses may be challenging, and contaminating individuals can be introduced accidentally. In this context, the identification of such contaminants is important to avoid compromising further selection cycles, as well as genetic and genomic studies. The main objective of this work was to propose an automated multivariate methodology for the detection and classification of putative contaminants, including apomictic clones, self-fertilized individuals, half-siblings and full contaminants, in biparental polyploid progenies of tropical forage grasses. We established a pipeline to identify contaminants in **genotyping-by-sequencing (GBS)** data encoded as **allele dosages of single nucleotide polymorphism (SNP)** markers by integrating **principal component analysis (PCA)**, genotypic analysis (GA) measures based on Mendelian segregation and **clustering analysis (CA)**. The combination of these methods allowed the correct identification of all contaminants in all simulated progenies and the detection of putative contaminants in three real progenies of tropical forage grasses, providing an easy and promising methodology for the identification of contaminants in biparental progenies of tetraploid and hexaploid species. The proposed pipeline was made available through the <a href="https://github.com/lagmunicamp/polycid" target="_blank">**polyCID**</a> Shiny app and can be easily coupled with traditional genetic approaches, such as linkage map construction, thereby increasing the efficiency of breeding programs.

    
* ### [Fraudulent Transaction Detection](https://github.com/felipebita/fraud_detection)
    Financial institutions are facing problems when dealing with fraudulent transactions. Those transactions not only reduces the institution revenue but also its credibility. In this scenario, the Block Fraud Company offers a service that ensures the blocking of fraud transactions. The methodology developed by the BFC is based on machine learning algorithms for classification, the implemented model has a precision of 0.961 +/- 0.021 and a recall of 0.729 +/- 0.026. Considering the company expansion strategy and the model capacity, for a set of 52.185 transactions, the service would generate a total revenue of $57.260.473.
    
## Complementary Education
* **[(2022)](https://drive.google.com/file/d/1ai6LwEgR57ljlLUYnnVNlXejyzdhYtuA/view) AWS Cloud Concepts.** Datacamp (2 hours).
* **[(2022)](https://drive.google.com/file/d/1aUCvCw1mN-ksYgQSeUOPcFEJ_od1Eajs/view?usp=sharing) Intermediate SQL Queries.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1Z8r9yxfP9AybRhsd3jN1H7vaB_pKk5Wj/view?usp=sharing) Statistics for Biologists in R.** IV GBMeeting - Encontro do Programa de P√≥s-Gradua√ß√£o em Gen√©tica e Biologia Molecular - IB/UNICAMP, Brasil (20 hours).
* **[(2021)](https://drive.google.com/file/d/1ZIkSSqdwMqHPLXytZ0NSW0feyNoTOYC5/view?usp=sharing) Statistical Genetics: Data Analysis for Molecular Breeding.** 11¬∫ Congresso Brasileiro de Melhoramento de Plantas - CBMP (3 hours).
* **[(2021)](https://drive.google.com/file/d/1J_To2U8-gZpB0dQmsQzgyrkFFhBPOulW/view?usp=sharing) Genomic Selection.** Conecta GEM, Brasil. (6h)
* **[(2021)](https://drive.google.com/file/d/1Kvevj-b6x5IHdW6dMhtx4A4O6vIuMLIC/view?usp=sharing) Python for Natural Language Processing (NLP): ICMC 50th Anniversary Edition.** Instituto de Ci√™ncias Matem√°ticas e de Computa√ß√£o, USP, Brasil (20 hours).
* **(2021) (Course Assistant) Introduction to Python Programming.** Grupo de Estudos em Gen√©tica e Melhoramento (GenMelhor), UFV, Brasil (16 hours).
* **[(2021)](https://drive.google.com/file/d/15iZMPb9eaEpiupDYq-whndwP1_rwULCC/view?usp=sharing) (Course Assistant) Introduction to Python Programming.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (16 hours).
* **[(2021)](https://drive.google.com/file/d/19EUJxcxSmca0JYPNQ3DkGtgAuT_ugQA0/view?usp=sharing) Introduction to Machine Learning in Python.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **(2021) Spring Workshop on Applied Quantitative Genetics for Plant Breeders.** University of Illinois at Urbana-Champaign, USA.
* **(2021) Tools for Genomics-Assisted Breeding in Polyploids: Workshop.** Development of a Community Resource, National Institute of Food and Agriculture, USDA.
* **[(2019)](https://drive.google.com/file/d/1fTUxNCU-H9V7-5lfQbLS4Ql7Lm4u47gY/view?usp=sharing) Polyploid Genetic Data Analysis: From Dosage Calling To Linkage.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (21 hours).
* **[(2018)](https://drive.google.com/file/d/1j9asuhRvHxS7O-qa6QXbE3ln6dP42Lm8/view?usp=sharing) Statistical Mechanisms and Practical Applications of Genome-Wide Associatio.** Funda√ß√£o de Estudos Agr√°rios Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **[(2018)](https://drive.google.com/file/d/19J92QdIDi-0fcrrW1SJWtYoeCXQts-37/view?usp=sharing) Prediction of Breeding Values in Autotetraploid Forages.** Empresa Brasileira de Pesquisa Agropecu√°ria, EMBRAPA, Brasil (32 hours).
* **[(2018)](https://drive.google.com/file/d/1HeGfL3v1L-ugDlQieDHvPHUKhUdcEh5o/view?usp=sharing) Python for Genomic Data Science by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2018)](https://drive.google.com/file/d/18OKYGD5EfB4wUOzSXNLKz9LX3kxYh280/view?usp=sharing) Getting and Cleaning Data by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2018)](https://drive.google.com/file/d/1BmGpqLcqb4swR7zJlhL0VPlF6Hc4ad9_/view?usp=sharing) R Programming by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2017)](https://drive.google.com/file/d/1PkApy5Gi8uClQx5Rq0lpLvsPpv8jLAVQ/view?usp=sharing) Bioinformatic Winter Course: Workshop.** Universidade Federal de S√£o Paulo, UNIFESP, Brasil (22 hours).
* **[(2017)](https://drive.google.com/file/d/1dtDY171JG3-n6YhZjtS0z5LyYtyUkg87/view?usp=sharing) Bioinformatic Analysis of NGS Data.** Sociedade Brasileira de Gen√©tica, SBG, Brasil (3 hours).

## Lectures and Presentations
* ### Contaminant Identification in Polyploids (videos)(en)
    [Video-poster](https://drive.google.com/file/d/1K8t9R4DzqFK9VmUHQcnjwUwpBBbcKFyv/view?usp=sharing) of the article [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full) submited in the [11¬∫ CBPM - Brazilian Congress of Plant Breeding](https://www.cbmp2021.com.br/).
    
    [Complete Presentation](https://www.youtube.com/watch?v=s2i3RN-3FJI) for the [IV GBMeeting - Encontro do Programa de P√≥s-Gradua√ß√£o em Gen√©tica e Biologia Molecular - IB/UNICAMP](https://gbmeetingunicamp.wixsite.com/gbmeeting), this work was awarded [second place](https://drive.google.com/file/d/1ZNiU0DJR6gp9UOYBPMgQW2FgjUXWRg8i/view?usp=sharing) in the Bioinformatics section.
 
* ### Bioinformatics and Artificial Inteligence applied to Genomics [(slides)](https://docs.google.com/presentation/d/1hUDR54jrX8WfxMptwUonaxQKousb_t6eU38vOSNbQvI/edit?usp=sharing)(en)
    This slide presentation was used in a lecture ministered in the [BV782 - Plant Genetics and Genomics - 1S/2022](https://www.dac.unicamp.br/portal/caderno-de-horarios/2022/1/S/G/IB/BV782) class for the Biological Sciences course at UNICAMP. It shows a historical view, importance, application and future of the Bioinformatics field and a introduction to Artificial Inteligence, mostly about Machine Learning techniques, including supervised (regression and classification) and unsupervised (clustering and dimensionality reduction) learning and its metrics of evaluation, with examples of application in genetics of plants.
   
* ### Genomic Selection [(video)](https://drive.google.com/file/d/1mCsgztu_3Aw5fAifUz7Rt2LNbj2TSLjq/view?usp=sharing)(pt-br)
    This lecture was ministered in the [BV782 - Plant Genetics and Genomics - 1S/2020](https://www.dac.unicamp.br/portal/caderno-de-horarios/2020/1/S/G/IB/BV782) class for the Biological Sciences course at UNICAMP. It is an introduction about Genomic Selection, passing through the famous Meuwissen (2001) article which demonstrate the applicability of dense sets of genetic markers to predict breeding values, how Genomic Selection models are evaluted (Predictive Ability and Cross Validation) and part of my research regarding genomic selection in half-siblings families of *Urochloa ruziziensis* tropical forage.
    
* ### Genetic Markers [(slides)](https://drive.google.com/file/d/1jXof4E6BPAY00_-0V_jWP3NAH1zdsNeK/view?usp=sharing)(pt-br)
    This slide presentation was used in a lecture ministered in the [BV880 - Plant Breeding - 2S/2019](https://www.dac.unicamp.br/portal/caderno-de-horarios/2019/2/S/G/IB/BV880) class for the Biological Sciences course at UNICAMP. It is an introduction about markers (morphological and molecular) with focus on molecular markers,especially the DNA markers. It covers the most common markers (microsatellite and SNP) and it's use in linkage maps and QTL mapping.

* ### Linear Regression [(slides)](https://docs.google.com/presentation/d/1R6fViHULsb0LGOy7FC9hUny7v4ktDfaD/edit?usp=sharing&ouid=106210785402829867900&rtpof=true&sd=true)(en)
    This presentation was constructed as a scheme for my own studies and latter used to teach lab colleagues. It covers the three main topics about Linear Regressions: (1) fit a line to the data; (2) calculate the R¬≤ and (3) calculate de F-test.    

## Working Setup

        Desktop                                                                                Laptop                
        Motherboard: Aorus Z370M                                                               Apple MacBook Pro: 13in 2017
        CPU: Intel Core i5 8400                                                                CPU: Intel Core i5 7360U
        RAM: Corsair Vengeance LPX 8GB (2x4GB) 3000Mhz                                         RAM: LPDDR3 8 GB 2133 MHz
        Disk: SSD Kingston 2.5¬¥ 240GB A400; Seagate Exp. Port. Drive 2TB                       Disk: 256 GB SSD 
        Graphics: Asus GeForce GTX 1060 OC 6GB GDDR5 Rog Strix 192-BIT
        Monitor: 24in 144hz Acer GN246HL; 24in IPS Philips 242V8A
        Keyboard: HyperX Mars
        Mouse: Logitech G403H
        Microphone: Blue Snowball
</div>

<!--
**felipebita/felipebita** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
