<h1 align="center">Felipe Martins, PhD</h1>

<h3 align="center"><i>Data Scientist and Bioinformatician</i></h3>

<div align="center">

[![Linkedin Badge](https://img.shields.io/badge/LinkedIn-0077B5?style=flat&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/febimartins/)&nbsp;&nbsp;
[![Medium Badge](https://img.shields.io/badge/Medium-12100E?style=flat&logo=medium&logoColor=white)](https://medium.com/@febimartins)&nbsp;&nbsp;
[![Twitter Badge](https://img.shields.io/badge/Twitter-1DA1F2?style=flat&logo=twitter&logoColor=white)](https://twitter.com/Felitencourt)&nbsp;&nbsp;
[![Instagram Badge](https://img.shields.io/badge/Instagram-E4405F?style=flat&logo=instagram&logoColor=white)](https://www.instagram.com/bitafelipe/)&nbsp;&nbsp;
[![Gmail Badge](https://img.shields.io/badge/Gmail-D14836?style=flat&logo=gmail&logoColor=white&link=mailto:febimartins@gmail.com)](mailto:febimartins@gmail.com)&nbsp;&nbsp;
</div>

## About Me
Analytical and investigative profile with good teamwork skills. I have six years of experience in data analysis for genetics/plant breeding projects. During this period, I have gained skills in programming (R and Python), statistical analysis/modeling, machine learning, project development, technical writing, and storytelling. In addition to these skills, I have knowledge of Spark, SQL, GitHub, and PowerBI. I am currently transitioning my career to fields related to business, finance and banking, I have projects on fraud detection, credit risk modeling e sales analysis. My portfolio (on Git) contains a complete description of my academic projects, study projects, certifications, and lectures/presentations. 

Curriculum vitae [(EN)](https://drive.google.com/file/d/1e0tLglt8cKbr7wknGx_VrKM4DCiBZ8ch/view?usp=sharing)  [(PTBR)](https://drive.google.com/file/d/1djEAULOTYeq2MUAB_FCemdrv0SvCXmYa/view?usp=sharing)

## Skills
        Data Science                                     Research                                           Microsoft Windows
        Data Analysis                                    Bioinformatics                                     Linux
        Data Mining                                      Computational Biology                              Apple Mac
        Data Visualization                               Genetics                                           Ubuntu
        Statistics                                       Plant Genetics                                     CentOS
        Machine Learning                                 Plant Breeding                                     
        Python                                           Genomic Data
        R                                                Genotyping
        SQL                                              Molecular Markers
        GitHub                                           GBS
        AWS                                              Transcriptome
        Spark                                            Simulation
        Microsoft Office/Power BI                        Teaching

## Education
* **2018 - 2022**
    * **Doctorate in Genetics and Molecular Biology - Bioinformatics** [:arrow_forward:](https://drive.google.com/file/d/1eFqxvjoPIK_hLVHvzAQMOeChNmGcpaj4/view?usp=sharing)
    * State University of Campinas - Unicamp - Brazil 
    * Laboratory of Genetic and Molecular Analysis [:arrow_forward:](https://lagm.cbmeg.unicamp.br/en/index.html)
    * Title: Contaminant Identification And Multi-omic Analysis Applied In Polyploid Tropical Forage Grasses Molecular Breeding [:arrow_forward:](https://drive.google.com/file/d/1dl5jDmfgOPFXDuvZrtiKm5g9yIiE0QXt/view?usp=sharing)
    * Supervisor: Anete Pereira de Souza [:arrow_forward:](https://www.linkedin.com/in/anete-pereira-de-souza-40728b67/)
                     
* **2014 - 2017**
    * **Bachelor in Biological Sciences - Bioinformatics** [:arrow_forward:](https://drive.google.com/file/d/1pI8eVAobot_1tvGbaeYL_SpZDoFVsdbc/view?usp=sharing)
    * State University of Campinas - Unicamp - Brazil
    * Laboratory of Genetic and Molecular Analysis [:arrow_forward:](https://lagm.cbmeg.unicamp.br/en/index.html)
    * Title: Identification And Characterization Of Single Copy Candidate Genes In Sugarcane
    * Supervisor: Anete Pereira de Souza [:arrow_forward:](https://www.linkedin.com/in/anete-pereira-de-souza-40728b67/) 
                                   
## Scientific Production and Projects

* ### [Genome-wide Family Prediction Unveils Molecular Mechanisms Underlying the Regulation of Agronomic Traits in Urochloa ruziziensis](https://www.biorxiv.org/content/10.1101/2023.09.25.559305v1) [(Flowchart)](https://drive.google.com/file/d/1EQoMBI_S97q8Ij12aUPFb-UAPekSEIcZ/view?usp=sharing) (ML, Networks, R, Python)
 
    Tropical forage grasses, especially species of the genus Urochloa, play an important role in cattle production and are the main food source for animals in tropical/subtropical regions. Most of the species are apomictic and tetraploid, which gives special importance to U. ruziziensis, a sexual diploid species that can be tetraploidized for use in interspecific crosses with apomictic species. As a means to assist in breeding programs, this study investigates the applicability of **genome-wide family prediction** (GWFP) in U. ruziziensis half-sibling families to predict growth and biomass production. **Machine learning** and **feature selection** algorithms were used to reduce the necessary number of markers for prediction and to enhance the predictive ability across the phenotypes. Beyond that, to investigate the regulation of agronomic traits, the positions of the markers with more importance for the prediction were considered putatively associated to quantitative trait loci (QTLs), and in a **multiomic approach**, genes obtained in the species transcriptome were mapped and linked to those markers. Furthermore, a **gene coexpression network** was modeled, enabling the investigation of not only the mapped genes but also their coexpressed genes. The functional annotation showed that the mapped genes are mainly associated with auxin transport and biosynthesis of lignin, flavonol and folic acid, while the coexpressed genes are associated with DNA metabolism, stress response and circadian rhythm. The results provide a viable marker-assisted breeding approach for tropical forages and identify target regions for future molecular studies on these agronomic traits.
        
* ### [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full) (ML, R)

    Artificial hybridization plays a fundamental role in plant breeding programs since it generates new genotypic combinations that can result in desirable phenotypes.  Depending on the species and mode of reproduction, controlled crosses may be challenging, and contaminating individuals can be introduced accidentally. In this context, the identification of such contaminants is important to avoid compromising further selection cycles, as well as genetic and genomic studies. The main objective of this work was to propose an automated multivariate methodology for the detection and classification of putative contaminants, including apomictic clones, self-fertilized individuals, half-siblings and full contaminants, in biparental polyploid progenies of tropical forage grasses. We established a pipeline to identify contaminants in **genotyping-by-sequencing (GBS)** data encoded as **allele dosages of single nucleotide polymorphism (SNP)** markers by integrating **principal component analysis (PCA)**, genotypic analysis (GA) measures based on Mendelian segregation and **clustering analysis (CA)**. The combination of these methods allowed the correct identification of all contaminants in all simulated progenies and the detection of putative contaminants in three real progenies of tropical forage grasses, providing an easy and promising methodology for the identification of contaminants in biparental progenies of tetraploid and hexaploid species. The proposed pipeline was made available through the <a href="https://github.com/lagmunicamp/polycid" target="_blank">**polyCID**</a> Shiny app and can be easily coupled with traditional genetic approaches, such as linkage map construction, thereby increasing the efficiency of breeding programs.
    
* ### [Fraudulent Transaction Detection](https://github.com/felipebita/fraud_detection) (ML, Python)
    Financial institutions are facing problems when dealing with fraudulent transactions. Those transactions not only reduces the institution revenue but also its credibility. In this scenario, the Block Fraud Company offers a service that ensures the blocking of fraud transactions. The methodology developed by the BFC is based on machine learning algorithms for classification, the implemented model has a precision of 0.961 +/- 0.021 and a recall of 0.729 +/- 0.026. Considering the company expansion strategy and the model capacity, for a set of 52.185 transactions, the service would generate a total revenue of $57.260.473.
    
* ### [Credit Risk Modeling](https://github.com/felipebita/credit_risk/tree/main) (ML, Python) (NotFinished)
    Credit risk analysis is of paramount importance for financial institutions as it allows them to assess the probability of borrowers defaulting on their loan obligations. By accurately predicting credit risk, these institutions can make informed decisions regarding lending, determine appropriate interest rates, and maintain a balanced loan portfolio. Effective credit risk analysis not only minimizes financial losses but also contributes to the stability of the financial system as a whole.    
    
* ### [Digital Plataform Sales Data Analysis](https://github.com/felipebita/digital_market_sales) (SQL)
    FicName is a leading platform in the market for distribution and sale of digital products, offering a comprehensive infrastructure for digital entrepreneurs and content producers who wish to market online courses, books, e-books, videos, and other digital materials. In this report, I’m going to present the strategies, analyses, and findings achieved while working with a dataset, with the goal of highlighting my knowledge in data analysis. The objective is to provide valuable insights to support strategic decisions, reflecting the daily responsibilities of a data analyst in the organization. 
    
* ### [Optimizing Online Sports Retail Revenue](https://github.com/felipebita/optimizing_online_sports_retail_revenue) (SQL)
    In this SQL project, I had the opportunity to analyze product data for an online sports retail company. Working with pricing, revenue, ratings, reviews, descriptions, and website traffic data, I applied techniques such as aggregation, cleaning, labeling, Common Table Expressions, and correlation to optimize revenue. The project involved various tasks, including counting missing values, comparing pricing, labeling price ranges, evaluating average discounts by brand, analyzing the correlation between revenue and reviews, and investigating the effect of product description length on ratings and reviews. I also evaluated the performance of footwear and clothing products separately by exploring reviews by month and brand. Through this project, I was able to showcase my data analysis skills and gained valuable experience working with complex datasets.
   
## Complementary Education
* **[(2023)](https://drive.google.com/file/d/1bfEAmXhMG-I3-7fZ5Xibin3Nw2J2qE4n/view?usp=sharing) Case Study: Analyzing Customer Churn in Excel.** Datacamp (3 hours).
* **[(2023)](https://drive.google.com/file/d/1_0zBjlAuUKgMgylQ_HxG5cEL5Vd5Grio/view?usp=sharing) Data Analysis in Excel.** Datacamp (3 hours).
* **[(2023)](https://drive.google.com/file/d/1YBmJW-D9rLC8p7y3r02pH4fgwL782oUy/view?usp=sharing) Data Visualization in Excel.** Datacamp (3 hours).
* **[(2023)](https://drive.google.com/file/d/1SycPtxT4I5hf2ObVUY_CwGeW1WuqHYDF/view) Data Preparation in Excel.** Datacamp (3 hours).
* **[(2023)](https://drive.google.com/file/d/1Syb8_y4ZpRm4lhorvtqn_T6GOoMq1lt6/view) Introduction to Excel.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1I6tInUNcW5lWgpTLCI6BqNQ_Zf1Df5kR/view?usp=sharing) Web Scraping in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1GmYct5MRrJT8QT2TguQhStXsKtt1XhRa/view) Machine Learning for Time Series Data in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1FZwmjYGqaxuauh4aOThteQh0Klc-QiiS/view?usp=sharing) ARIMA models in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1FKzvD5ragVbRfd_8PpAiLhyLgaGTOIiC/view?usp=sharing) Time Series Analysis in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1FHrMj9aWCNNZyI-9DJiY14zz3xwWad5G/view?usp=sharing) Manipulating Time Series Data in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1fF3VtFfmJBp2n2ffkhBy1gl_ZID9-cTL/view?usp=sharing) Credit Risk Modeling in Python.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/103VFi3z0LpAzpDRTPs-gt-lttj9WQ7i0/view?usp=sharing) Fundamentals of Credit.** Corporate FInance Institute (2 hour).
* **[(2023)](https://drive.google.com/file/d/1fE8Co0DhhD5d7PjjF6Omssukq8QHg2ki/view?usp=sharing) Introduction to Fintech.** Corporate Finance Institute (1 hour).
* **[(2023)](https://drive.google.com/file/d/1fC8GL19hZNDbyOoZJTXCHxORk2YSSPKg/view?usp=sharing) Introduction to Banking.** Corporate Finance Institute (2 hours).
* **[(2023)](https://drive.google.com/file/d/1fC7reen0sIIOae-Ap_XH9LWZc8kyZZzt/view?usp=sharing) Data Manipulation in SQL.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1f2ZI09B8BYWaFT6ncFecinABF6czzGEh/view?usp=sharing) Joining Data in SQL.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1dZAEOB45hBkpO0tF-yGC5o1jy_0RlUet/view) Introduction to DAX in Power BI.** Datacamp (2 hours).
* **[(2023)](https://drive.google.com/file/d/1caLGzBtlliDOqo0UdfOKmIMgU4bdytZt/view) Introduction to Power BI.** Datacamp (3 hours).
* **[(2023)](https://drive.google.com/file/d/1dkrZnh_2qZGO4s10hefiMNRckbPEnuN_/view?usp=sharing) Feature Engineering with PySpark.** Datacamp (4 hours).
* **[(2023)](https://drive.google.com/file/d/1c_2EdtVxgXGfd70KhM7FDPNFSfVsYz1q/view?usp=sharing) Cleaning Data with PySpark.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1b7ohhfAYRzKhiNE2pfllhxDBx4mCD4Ly/view?usp=sharing) Big Data Fundamentals with PySpark.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1b7bKvgzZeCIG-_RekUcwUzVasjJUikP0/view?usp=sharing) Introduction to PySpark.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1b94YesXczVIrJQ1ByNSSdeoHbBEgg51m/view?usp=sharing) Intermediate SQL.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1aUCvCw1mN-ksYgQSeUOPcFEJ_od1Eajs/view?usp=sharing) Intermediate SQL Queries.** Datacamp (4 hours).
* **[(2022)](https://drive.google.com/file/d/1b8m1JsFtVsFa_dYwS8bIQVJv4c2HRCT2/view?usp=sharing) Introduction to SQL.** Datacamp (2 hours).
* **[(2022)](https://drive.google.com/file/d/1ai6LwEgR57ljlLUYnnVNlXejyzdhYtuA/view) AWS Cloud Concepts.** Datacamp (2 hours).
* **[(2022)](https://drive.google.com/file/d/1Z8r9yxfP9AybRhsd3jN1H7vaB_pKk5Wj/view?usp=sharing) Statistics for Biologists in R.** IV GBMeeting - Encontro do Programa de Pós-Graduação em Genética e Biologia Molecular - IB/UNICAMP, Brasil (20 hours).
* **[(2021)](https://drive.google.com/file/d/1ZIkSSqdwMqHPLXytZ0NSW0feyNoTOYC5/view?usp=sharing) Statistical Genetics: Data Analysis for Molecular Breeding.** 11º Congresso Brasileiro de Melhoramento de Plantas - CBMP (3 hours).
* **[(2021)](https://drive.google.com/file/d/1J_To2U8-gZpB0dQmsQzgyrkFFhBPOulW/view?usp=sharing) Genomic Selection.** Conecta GEM, Brasil. (6h).
* **[(2021)](https://drive.google.com/file/d/1Kvevj-b6x5IHdW6dMhtx4A4O6vIuMLIC/view?usp=sharing) Python for Natural Language Processing (NLP): ICMC 50th Anniversary Edition.** Instituto de Ciências Matemáticas e de Computação, USP, Brasil (20 hours).
* **(2021) (Course Assistant) Introduction to Python Programming.** Grupo de Estudos em Genética e Melhoramento (GenMelhor), UFV, Brasil (16 hours).
* **[(2021)](https://drive.google.com/file/d/15iZMPb9eaEpiupDYq-whndwP1_rwULCC/view?usp=sharing) (Course Assistant) Introduction to Python Programming.** Fundação de Estudos Agrários Luiz de Queiroz, FEALQ, Brasil (16 hours).
* **[(2021)](https://drive.google.com/file/d/19EUJxcxSmca0JYPNQ3DkGtgAuT_ugQA0/view?usp=sharing) Introduction to Machine Learning in Python.** Fundação de Estudos Agrários Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **(2021) Spring Workshop on Applied Quantitative Genetics for Plant Breeders.** University of Illinois at Urbana-Champaign, USA.
* **(2021) Tools for Genomics-Assisted Breeding in Polyploids: Workshop.** Development of a Community Resource, National Institute of Food and Agriculture, USDA.
* **[(2019)](https://drive.google.com/file/d/1fTUxNCU-H9V7-5lfQbLS4Ql7Lm4u47gY/view?usp=sharing) Polyploid Genetic Data Analysis: From Dosage Calling To Linkage.** Fundação de Estudos Agrários Luiz de Queiroz, FEALQ, Brasil (21 hours).
* **[(2018)](https://drive.google.com/file/d/1j9asuhRvHxS7O-qa6QXbE3ln6dP42Lm8/view?usp=sharing) Statistical Mechanisms and Practical Applications of Genome-Wide Associatio.** Fundação de Estudos Agrários Luiz de Queiroz, FEALQ, Brasil (8 hours).
* **[(2018)](https://drive.google.com/file/d/19J92QdIDi-0fcrrW1SJWtYoeCXQts-37/view?usp=sharing) Prediction of Breeding Values in Autotetraploid Forages.** Empresa Brasileira de Pesquisa Agropecuária, EMBRAPA, Brasil (32 hours).
* **[(2018)](https://drive.google.com/file/d/1HeGfL3v1L-ugDlQieDHvPHUKhUdcEh5o/view?usp=sharing) Python for Genomic Data Science by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2018)](https://drive.google.com/file/d/18OKYGD5EfB4wUOzSXNLKz9LX3kxYh280/view?usp=sharing) Getting and Cleaning Data by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2018)](https://drive.google.com/file/d/1BmGpqLcqb4swR7zJlhL0VPlF6Hc4ad9_/view?usp=sharing) R Programming by Johns Hopkins University on Coursera.** Johns Hopkins University, JHU, Estados Unidos.
* **[(2017)](https://drive.google.com/file/d/1PkApy5Gi8uClQx5Rq0lpLvsPpv8jLAVQ/view?usp=sharing) Bioinformatic Winter Course: Workshop.** Universidade Federal de São Paulo, UNIFESP, Brasil (22 hours).
* **[(2017)](https://drive.google.com/file/d/1dtDY171JG3-n6YhZjtS0z5LyYtyUkg87/view?usp=sharing) Bioinformatic Analysis of NGS Data.** Sociedade Brasileira de Genética, SBG, Brasil (3 hours).

## Lectures and Presentations
* ### PhD Defense Presentation [(slides)](https://drive.google.com/file/d/1avUpwH16_cMy_DZk7N3cYd0LzD9HfED8/view?usp=sharing)(en)
    Contaminant identification and multiomic analysis applied in polyploid tropical forage grasses molecular breeding 
    

* ### Contaminant Identification in Polyploids (videos)(en)
    [Video-poster](https://drive.google.com/file/d/1K8t9R4DzqFK9VmUHQcnjwUwpBBbcKFyv/view?usp=sharing) of the article [A Semi-Automated SNP-Based Approach for Contaminant Identification in Biparental Polyploid Populations of Tropical Forage Grasses](https://www.frontiersin.org/articles/10.3389/fpls.2021.737919/full) submited in the [11º CBPM - Brazilian Congress of Plant Breeding](https://www.cbmp2021.com.br/).
    
    [Complete Presentation](https://www.youtube.com/watch?v=s2i3RN-3FJI) for the [IV GBMeeting - Encontro do Programa de Pós-Graduação em Genética e Biologia Molecular - IB/UNICAMP](https://gbmeetingunicamp.wixsite.com/gbmeeting), this work was awarded [second place](https://drive.google.com/file/d/1ZNiU0DJR6gp9UOYBPMgQW2FgjUXWRg8i/view?usp=sharing) in the Bioinformatics section.
 
* ### Bioinformatics and Artificial Inteligence applied to Genomics [(slides)](https://docs.google.com/presentation/d/1hUDR54jrX8WfxMptwUonaxQKousb_t6eU38vOSNbQvI/edit?usp=sharing)(en)
    This slide presentation was used in a lecture ministered in the [BV782 - Plant Genetics and Genomics - 1S/2022](https://www.dac.unicamp.br/portal/caderno-de-horarios/2022/1/S/G/IB/BV782) class for the Biological Sciences course at UNICAMP. It shows a historical view, importance, application and future of the Bioinformatics field and a introduction to Artificial Inteligence, mostly about Machine Learning techniques, including supervised (regression and classification) and unsupervised (clustering and dimensionality reduction) learning and its metrics of evaluation, with examples of application in genetics of plants.
   
* ### Genomic Selection [(video)](https://drive.google.com/file/d/1mCsgztu_3Aw5fAifUz7Rt2LNbj2TSLjq/view?usp=sharing)(pt-br)
    This lecture was ministered in the [BV782 - Plant Genetics and Genomics - 1S/2020](https://www.dac.unicamp.br/portal/caderno-de-horarios/2020/1/S/G/IB/BV782) class for the Biological Sciences course at UNICAMP. It is an introduction about Genomic Selection, passing through the famous Meuwissen (2001) article which demonstrate the applicability of dense sets of genetic markers to predict breeding values, how Genomic Selection models are evaluted (Predictive Ability and Cross Validation) and part of my research regarding genomic selection in half-siblings families of *Urochloa ruziziensis* tropical forage.
    
* ### Genetic Markers [(slides)](https://drive.google.com/file/d/1jXof4E6BPAY00_-0V_jWP3NAH1zdsNeK/view?usp=sharing)(pt-br)
    This slide presentation was used in a lecture ministered in the [BV880 - Plant Breeding - 2S/2019](https://www.dac.unicamp.br/portal/caderno-de-horarios/2019/2/S/G/IB/BV880) class for the Biological Sciences course at UNICAMP. It is an introduction about markers (morphological and molecular) with focus on molecular markers,especially the DNA markers. It covers the most common markers (microsatellite and SNP) and it's use in linkage maps and QTL mapping.

* ### Linear Regression [(slides)](https://docs.google.com/presentation/d/1R6fViHULsb0LGOy7FC9hUny7v4ktDfaD/edit?usp=sharing&ouid=106210785402829867900&rtpof=true&sd=true)(en)
    This presentation was constructed as a scheme for my own studies and latter used to teach lab colleagues. It covers the three main topics about Linear Regressions: (1) fit a line to the data; (2) calculate the R² and (3) calculate de F-test.    

## Working Setup

        Desktop                                                                                Laptop                
        Motherboard: Aorus Z370M                                                               Apple MacBook Pro: 13in 2017
        CPU: Intel Core i5 8400                                                                CPU: Intel Core i5 7360U
        RAM: Corsair Vengeance LPX 8GB (2x4GB) 3000Mhz                                         RAM: LPDDR3 8 GB 2133 MHz
        Disk: SSD Kingston 2.5´ 240GB A400; Seagate Exp. Port. Drive 2TB                       Disk: 256 GB SSD 
        Graphics: Asus GeForce GTX 1060 OC 6GB GDDR5 Rog Strix 192-BIT
        Monitor: 24in 144hz Acer GN246HL; 24in IPS Philips 242V8A
        Keyboard: HyperX Mars
        Mouse: Logitech G403H
        Microphone: Blue Snowball
</div>

<!--
**felipebita/felipebita** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- 🔭 I’m currently working on ...
- 🌱 I’m currently learning ...
- 👯 I’m looking to collaborate on ...
- 🤔 I’m looking for help with ...
- 💬 Ask me about ...
- 📫 How to reach me: ...
- 😄 Pronouns: ...
- ⚡ Fun fact: ...
-->
